server/src/main/java/org/apache/druid/segment/realtime/appenderator/StreamAppenderator.java

```diff
@@ -1108,12 +1122,12 @@ SinkSchemaAnnouncer getSinkSchemaAnnouncer()
   /**
    * Unannounces the given base segment and all its upgraded versions.
```

Comment:
Please add a line about the returned set. Does it contain the baseIdentifier too?

Reply:
Done